packages/charts/src/vaadin-chart-series-mixin.js

```diff
@@ -0,0 +1,400 @@
+/**
+ * @license
+ * Copyright (c) 2000 - 2025 Vaadin Ltd.
+ *
+ * This program is available under Vaadin Commercial License and Service Terms.
+ *
+ *
+ * See https://vaadin.com/commercial-license-and-service-terms for the full
+ * license.
+ */
+
```

Comment:
nit: extra newline before import can be removed.

Reply:
Done